sdk/storage/Azure.Storage.Files.DataLake/src/Models/DataLakeFileReadResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Storage.Files.DataLake.Models
+{
+    /// <summary>
+    /// The details and content returned from reading a DataLake File.
+    /// </summary>
+    public class DataLakeFileReadResult
```

Comment:
We need to add methods to create `DataLakeFileReadResult` and `DataLakeFileReadStreamingResult` to [DataLakeModelFactory](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Files.DataLake/src/Models/DataLakeModelFactory.cs) .

Reply:
Thanks for the review, will add this.
I'll also add more tests and update the Samples